src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -199,6 +199,12 @@ void handleDirectCommand(Channel channel, User sender, String payload) {
             return;
         }
 
+        m = Pattern.compile("archive (?:github )repo (\\S+) (\\S+)",CASE_INSENSITIVE).matcher(payload);
```

Comment:
I don't think this will work for no description matched, as it requires a space and thing a non empty string